aws/resource_aws_elasticache_cluster.go

```diff
@@ -180,6 +180,9 @@ func resourceAwsElasticacheCluster() *schema.Resource {
 		ForceNew: true,
 	}
 
+	resourceSchema["availability_zones"].ConflictsWith = []string{"preferred_availability_zones"}
```

Comment:
I have created #4828 to fully expand the resource schemas rather than continue with this generated approach.